packages/@lwc/babel-plugin-component/src/types.ts

```diff
@@ -22,6 +22,7 @@ export interface LwcBabelPluginOptions {
     instrumentation?: InstrumentationObject;
     apiVersion?: number;
     enableSyntheticElementInternals?: boolean;
+    featureFlag?: string;
```

Comment:
```suggestion
    componentFeatureFlagModule?: string;
```
`featureFlag` could be very easily confused with `@lwc/features` feature flags, so we should probably call it something else.